pom.xml

```diff
@@ -78,7 +44,7 @@
 							<outputDirectory>${basedir}/target/OpenHospital20/rsc</outputDirectory>
 							<resources>
 								<resource>
-									<directory>rsc</directory>
+									<directory>src/main/resources</directory>
```

Comment:
I suspect this can be omitted, since that's the standard location of resources: http://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html